site/kubernetes/operator/using-operator.md

```diff
@@ -940,6 +941,88 @@ For more information about concepts mentioned above, see:
+* `myrabbit-server-1.myrabbit-nodes.mynamespace.svc.cluster.local`
+* `myrabbit-server-2.myrabbit-nodes.mynamespace.svc.cluster.local`
+* `myrabbit.mynamespace.svc.cluster.local`
+
+Note that the last SAN attribute is the client service DNS name.
+Depending on the service type used (`spec.service.type`), further SAN attributes may be required.
+For example, if using service type `NodePort`, the SAN must include the external IP address of each Kubernetes node.
+
+To enable TLS, create a Kubernetes secret containing the pem encoded server certificate `server.pem` and private key `server-key.pem`
+
+<pre class='lang-bash'>
+kubectl create secret tls tls-secret --cert=server.pem --key=server-key.pem
+</pre>
+
+or use a tool such as <a href="https://cert-manager.io/">Cert Manger</a> to generate a tls secret.
+
+Once this secret exists, a RabbitMQ cluster can be deployed following the <a href="https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/tls">TLS example</a>.
+
+<pre class="lang-yaml">
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: additional-port
+spec:
+  replicas: 1
+  tls:
+    secretName: tls-secret
+</pre>
+
+### <a id='mutual-tls' class='anchor' href='#mutual-tls'>Mutual TLS encryption between clients and RabbitMQ</a>
+
+Mutual TLS (mTLS) enhances TLS by requiring that the server verify the identity of the client, in addition to the client verifying the server, which already occurs in TLS encryption. In order for this mutual verification to occur, both the client and server must be configured with certificate and key pairs, with the client pair signed by a CA trusted by the server and the server pair signed by a CA trusted by the client. The mutual verification process is shown in the following diagram:
+
+<img src="/img/mTLS.png"/>
+
+In addition to the <a href="#one-way-tls">configuration required to support TLS</a>, configuring mutual TLS requires the RabbitMQ cluster to be configured with the CA certificate used to sign the client certificate and key pair, `ca.pem`. Create a Kuberntes secret with key `ca.crt` containing this secret
+
+<pre class='lang-bash'>
+kubectl create secret generic ca-secret --from-file=ca.crt=ca.pem
+</pre>
+
+or create this secret using a tool such as <a href="https://cert-manager.io/">Cert Manager</a>.
+
+Once this secret and the `tls-secret` exist, a RabbitMQ cluster cluster can be deployed following the <a href="https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/mtls">mTLS example</a>.
+
+<pre class="lang-yaml">
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: mtls
+spec:
+  replicas: 1
+  tls:
+    secretName: tls-secret
+    caSecretName: ca-secret
+</pre>
+
+In order to enforce client verification, RabbitMQ must be configured to reject clients that do not present certificates. This can be done by enabling `ssl.options.fail_if_no_peer_cert` in the additional config:
```

Comment:
`ssl.options` should be `ssl_options`. We should also link to [this TLS guide section](https://www.rabbitmq.com/ssl.html#peer-verification), most users have a fairly vague idea of what peer verification really is.